scripts/simplify_network.py

```diff
@@ -620,6 +714,10 @@ def consense(x):
 
     update_p_nom_max(n)
 
+    e_threshold_drop_isolated = cluster_config.get("e_threshold_drop_isolated", False)
+    if e_threshold_drop_isolated:
+        drop_isolated_nodes(n, drop_e_threshold=e_threshold_drop_isolated)
```

Comment:
Ah little refuse: here we need n = drop_isolated_nodes(...)

Reply:
Let's try :)
Although, I have an impression that is has worked in the previous version, too
At least, the isolated buses disappeared from the network picture

Reply:
Oh yeah, there may not be the need for n = but later in the merge nodes there will be most likely.
Let's leep it :) it doesn't harm

Reply:
I have an impression that it was a change that made CI work...